src/en/news/crimson/index.md

```diff
@@ -0,0 +1,129 @@
+---
+layout: content-simple
+title: Crimson Project
+order: 4
+---
+
+# {{ title }}
+
+## Goal
+
+Crimson is designed to be a faster OSD, in the sense that
+
+- It targets fast storage devices, like NVMe storage, to take advantage of the high performance of random I/O and high throughput of the new hardware.
+- It will be able to bypass the kernel when talking to networking/storage devices which support polling.
+- It will be more computationally efficient, so the load is more balanced in a multi-core system, and each core will be able to serve the same amount of IO with lower CPU usage.
+- It will achieve these goals by a design from scratch. It also utilizes modern techniques, like SPDK, DPDK and the Seastar framework, so it can 1) bypass the kernel, 2) avoid memcpy between, 3) avoid lock contention.
+
+Crismon will be a drop-in-replacement of the classic OSD. Our long-term goal is to redesign the object storage backend from scratch, This new backend is dubbed “Seastore”. The disk layout we will use for Seastore won’t necessarily be optimal for HDDs. But Seastore is still in its early stage, and is still missing a detailed design. As an intermediate solution, bluestore is adapted to crimson-osd for testing purposes. But if the new object storage backend does not work well with HDD, we will continue using bluestore in crimson just for supporting HDD.
```

Comment:
I'll just make an editorial pass over this after this PR is merged. Ignore these suggestions. I'll take care of them in the future.

Reply:
Merged, thanks!